code/modules/gear_presets/corpses.dm

```diff
@@ -310,14 +249,11 @@
 		ACCESS_CIVILIAN_COMMAND,
 	)
 
-/datum/equipment_preset/corpse/security/marshal/load_gear(mob/living/carbon/human/new_human)
-
-	new_human.equip_to_slot_or_del(new /obj/item/device/radio/headset/distress(new_human), WEAR_L_EAR)
+/datum/equipment_preset/corpse/security/cmb/load_gear(mob/living/carbon/human/new_human)
+	new_human.equip_to_slot_or_del(new /obj/item/device/radio/headset/distress/CMB/limited(new_human), WEAR_L_EAR)
 	new_human.equip_to_slot_or_del(new /obj/item/clothing/under/CM_uniform(new_human), WEAR_BODY)
 	if(SSmapping.configs[GROUND_MAP].environment_traits[MAP_COLD])
```

Comment:
No longer necessary to have this if check; you can just always `add_ice_colony_survivor_equipment(new_human)` because it has its own test for the MAP_COLD trait

Reply:
Done